website/content/docs/plugins/index.mdx

```diff
@@ -0,0 +1,200 @@
+  the core source code. Packer plugins are able to add new builders,
+  provisioners, hooks, and more.
+page_title: Plugins
+sidebar_title: Packer Plugins
+---
+
+# Packer Plugins
+
+Packer Plugins allow new functionality to be added to Packer without modifying
+the core source code. Packer plugins are able to add new builders, provisioners,
+hooks, and more. In fact, much of Packer itself is implemented by writing
+plugins that are simply distributed with Packer. For example, many of the
+builders, provisioners, and more that ship with Packer are implemented as
+Plugins that are simply hardcoded to load with Packer.
+
+* This page documents how to install plugins.
+* Plugins cannot be configured yet.
+* If you're interested in developing plugins, see the [developing 
+plugins](/docs/plugins/creation#developing-plugins) page.
+
+The current official listing of available Packer plugins can be found
+[here](/community-tools#third-party-plugins). This is an incomplete list, and
+more plugins can be found by searching. Typically, searching "packer plugin _x_"
+will find what you're looking for if it exists. We hope to create an offical
+registry for third party plugins in the future.
+
+## How Plugins Work
+
+Packer plugins are completely separate, standalone applications that the core
+of Packer starts and communicates with.
+
+These plugin applications aren't meant to be run manually. Instead, Packer core
+executes them as a sub-process, run as a sub-command (`packer plugin`) and
+communicates with them. For example, the Shell provisioner is actually run as
+`packer plugin packer-provisioner-shell`. The next time you run a Packer build,
+look at your process list and you should see a handful of `packer-` prefixed
+applications running.
+
+
+<Tabs>
+<Tab heading="Plugins installed with Packer init (from Packer v1.7.0)">
+
+~> **Note**: Only _multi-plugin binaries_ -- that is plugins named
+packer-plugin-*, like the `packer-plugin-amazon` -- are expected to work with
+Packer init. The legacy `builder`, `post-processor` and `provisioner` plugin
+types will keep on being detected but Packer cannot install them automatically.
+
+## Installing Plugins
+
+1. Add a
+[`required_plugin`](/docs/templates/hcl_templates/blocks/packer#specifying-plugin-requirements)
+block to your `packer` block, each block will tell Packer what version(s) can be
+installed, make sure to set a correct [version constraint
+string](/docs/templates/hcl_templates/blocks/packer#version-constraints). 
+  Example `required_plugins` block :
+
+  ```hcl
+  packer {
+    required_plugins {
+      myawesomecloud = {
+        version = ">= 2.7.0"
+        source = "azr/myawesomecloud"
+      }
+      otherawesomecloud = ">= 2.7.0"
+    }
+  }
+  ```
+
+2. Run [`packer init`](/docs/commands/init) to install all missing plugin binaries.
+
+## Names and Addresses
+Each plugin has two identifiers:
+
+* A `source` address, which is only used when requiring a plugin.
```

Comment:
Maybe something like
```suggestion
* A `source` address, which is only used when requiring a plugin outside the HashiCorp domain.
```

Reply:
If not was suggested, I think this sentence should be different because with or without `source` the plugin is under the required_plugins block so it's requiring a plugin.